src/codewhisperer/service/keyStrokeHandler.ts

```diff
@@ -88,17 +86,27 @@ export class KeyStrokeHandler {
                 }
                 case DocumentChangedSource.IntelliSense: {
                     this.keyStrokeCount += 1
-                    triggerType = 'IntelliSenseAcceptance'
+                    if (duration >= CodeWhispererConstants.invocationTimeIntervalThreshold) {
+                        triggerType = 'IntelliSenseAcceptance'
+                    }
```

Comment:
actually the time threhold should not be applied to IntelliSense trigger I believe? At least this is the case for JB IIRC.

Reply:
Done